plugins/vcenter_logs.yaml

```diff
@@ -0,0 +1,70 @@
+version: 0.0.1
+title: VMware vCenter
+description: Log parser for VMware vCenter
+parameters:
+  - name: listen_port
+    type: int
+    default: 5142
```

Comment:
I'm assuming this was intentionally changed from the stanza plugin?

Reply:
Yes, for the sake of testing this plugin, this was the port to point to for the tcp receive logs from the vCenter box I had loaded up, might be worth going back not entirely sure

Reply:
@armstrmi ok, so has the default for these logs changed from whenever until today? just making sure we are picking a good default that makes sense.

Reply:
I understand where you're coming from, looking back now it probably is best to change that back to the original port